### PR TITLE
587 tutorial "Using Covalent with PennyLane for hybrid computation"

### PR DESCRIPTION
**Pull request by ruihao-li**
Tutorial on a simple demonstration of integrating Covalent with PennyLane.
Link to issue: https://github.com/AgnostiqHQ/covalent/issues/587

- [x] I have added the tests to cover my changes.
- [x] I have updated the documentation and CHANGELOG accordingly.
- [x] I have read the CONTRIBUTING document.

### CODE REVIEW

**Comment by wjcunningham7** [comment]
Hello. You may have forgotten to update the changelog!
Please edit [`CHANGELOG.md`](/AgnostiqHQ/covalent/blob/master/CHANGELOG.md) with a one-to-two sentence description of the change in the UNRELEASED section. You may include a small working example for new features.

**Comment by jackbaker1001** [comment]
Hey @ruihao-li. I just took a look through this. It looks like a really good job to me, so well done!

I have one or two minor comments to make though:

(1) The hybrid computation that is being made is three-fold. (i) Quantum (photonic chip) (ii) Quantum (standard gate-model) and (iii) Classical compute. I then think that the classical compute should be it's own electron (i.e, the difference squared function).

(2) Could you please mention somewhere in a markdown cell that Covalent is able to intelligently schedule the tasks from all three paradigms? Right now we show how you would do it but do not motivate the use of covalent!

**Comment by ruihao-li** [comment]
Thanks @jackbaker1001! I made some modifications based on your comments. Please see the latest commit and description.

**Comment by jackbaker1001** [comment]
@ruihao-li This all looks great now and has my approval.

Once the failed check issue is dealt with (@kessler-frost what is that?) we should be good to merge.

**Review by jackbaker1001 — DISMISSED** [review]

**Comment by kessler-frost** [comment]
Hey @jackbaker1001, I think @ruihao-li might've missed updating the `CHANGELOG.md` which will make the version check fail.

**Comment by ruihao-li** [comment]
Hey @kessler-frost, will do that soon. @jackbaker1001, where do we want to place this tutorial, Beginner or Advanced?

**Comment by jackbaker1001** [comment]
@ruihao-li Beginer for this one

**Review by jackbaker1001 — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [UNRELEASED]
 
+- Added new tutorial: Using Covalent with PennyLane for hybrid computation.
```
Comment (kessler-frost) [inline CHANGELOG.md:10]: You also might wanna put this under a `Docs` heading.

**Pull request merged**